README.md

```diff
@@ -104,6 +104,27 @@ swaggen generate http://myapi.com/spec --template Templates/Swift  --destination
 
 For the Swift template, a handy option is `name`, which changes the name of the generated framework from the default of `API`. This can be set in the template or by passing in `--option name:MyCoolAPI`.
 
+
+### Option
+
+List of all available options
+
+name | action | expected values | default value
+--- | --- | --- | ---
+authors | change authors in podspec | `String` | Yonas Kolb
```

Comment:
Maybe we can leave out the `change...` from all of these. eg. `authors in podspec`

Reply:
ok